first.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio_Website</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+        integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+</head>
+
+<body>
+    <header>
+    <nav id="navbar">
+        <ul class="nav-list">
+            <li> <a href="#">about</a></li>
+            <li> <a href="#projects">work</a></li>
+            <li> <a href="#profile-link">contact</a></li>
+        </ul>
+    </nav>
+    </header>
+    <section id="welcome-section">
+        <h1>Hey we are Fen & Ouail</h1>
+        <p>Full Stack Web Developer</p>
+    </section>
+    <section id="projects" class="projet-tile">
+        <h2 class="projects-section-header">These are some of my projects</h2>
+        <div class="project_grid">
+            <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank"><img
+                    src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"></a>
+        </div>
```

Comment:
#alt img
Since you are using an img tag, for accessibility reasons, I suggest adding an alt text to it because sometimes due to poor network, images hardly will render.